numba/cuda/cudadrv/autotune.py

```diff
@@ -211,6 +214,23 @@ def prefer(self, *tpblist):
     'warp_alloc_gran': 4,
 })
 
+LIMITS_CC_70 = LIMITS_CC_62.copy()
+LIMITS_CC_70.update({
+    'smem_per_sm': SMEM96K,
+    'smem_per_block': SMEM96K,
+    'default_smem_config': SMEM96K,
+})
+
+LIMITS_CC_75 = LIMITS_CC_70.copy()
+LIMITS_CC_75.update({
+    'warp_per_sm': 32,
+    'thread_per_sm': 1024,
+    'block_per_sm': 16,
+    'smem_per_sm': SMEM64K,
+    'smem_per_block': SMEM64K,
+    'default_smem_config': SMEM64K,
+})
```

Comment:
May be not for now... but, it'd be really great to have a function that consumes one of these dictionaries and then prints out the values with the row labels as ordered in http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities such that it makes it easy to check (given there's a bunch of carried forward mutation across CC limits).

Reply:
It might be good to have - however, will this be deleted at some point in the future anyway, as autotuning is already deprecated? Would it be better to do something to accelerate the deletion of these dicts and the related code instead?

I skimmed the changes in the past, and it looks like the reason this is still here was as a fallback for the driver not supporting `cuOccpuancyMaxPotentialBlockSize`, and was deprecated back in v 0.40: 05e4a6c7fcc98e715f57be7b58a6dbc4f5e4b598. Do we know whether removing the autotuner now is likely to be a problem for present users of the latest version?

Reply:
good point, let's get this in as-is. Do you recall the minimum driver version for supporting the query functions?

@seibert assuming the driver level required for this feature is sufficiently old/common we ought to issue a proper deprecation notice tied to a version (perhaps 0.47 final support, 0.48 removed) or just remove it for 0.47, we've warned for 6 versions. Thoughts?

Reply:
cuOccupancyMaxPotentialBlockSize was present at least as far back as CUDA 8.0 (older CUDA docs don't seem to be readily accessible on the NVIDIA website), and we don't test Numba on anything older than CUDA 8 at this point.  I think it is safe to remove the fallback as we've deprecated it for a while.

Reply:
If you want to open a PR to remove this bit of code entirely, that would be fine.

Reply:
OK - I'll look into doing that, though if this PR's fixes could still go in in the meantime that would be great.. Then, post-0.47 I can have the removal ready.

Reply:
I think that is reasonable.  I will merge this as soon as our internal CI catches up.